### PR TITLE
added schema for new lcd config options

### PR DESCRIPTION
**Pull request by sumnerboy12**

### CODE REVIEW

File: src/OXRS_Rack32.cpp [hunk 1]
```diff
@@ -109,16 +109,45 @@ void _getConfigSchemaJson(JsonVariant json)
...
+  JsonObject inactiveBrightnessPercent = properties.createNestedObject("inactiveBrightnessPercent");
+  inactiveBrightnessPercent["title"] = "LCD Inactive Brightness (%)";
+  inactiveBrightnessPercent["description"] = "Brightness of the LCD when in-active (defaults to 10%). Must be a number between 0 and 100.";
+  inactiveBrightnessPercent["type"] = "integer";
+  inactiveBrightnessPercent["minimum"] = 0;
+  inactiveBrightnessPercent["maximum"] = 100;
+
+  JsonObject activeDisplaySeconds = properties.createNestedObject("activeDisplaySeconds");
+  activeDisplaySeconds["title"] = "LCD Active Display (seconds)";
+  activeDisplaySeconds["description"] = "How long the LCD remains 'active' after an event is detected (defaults to 10 seconds). Must be a number between 0 and 600 (i.e. 10 minutes).";
```
Comment (moinmoin-sh) [inline src/OXRS_Rack32.cpp:140]: Should be mentioned that 0 turns on forever (disables timer)

File: src/OXRS_Rack32.cpp [hunk 2]
```diff
@@ -109,16 +109,45 @@ void _getConfigSchemaJson(JsonVariant json)
...
+  activeDisplaySeconds["type"] = "integer";
+  activeDisplaySeconds["minimum"] = 0;
+  activeDisplaySeconds["maximum"] = 600;
+
+  JsonObject eventDisplaySeconds = properties.createNestedObject("eventDisplaySeconds");
+  eventDisplaySeconds["title"] = "LCD Event Display (seconds)";
+  eventDisplaySeconds["description"] = "How long the last event is displayed on the LCD (defaults to 3 seconds, setting to 0 disables the timeout so last event always shown). Must be a number between 0 and 600 (i.e. 10 minutes).";
```
Comment (moinmoin-sh) [inline src/OXRS_Rack32.cpp:147]: Should be mentioned that 0 turns on forever (disables timer)

**Review by moinmoin-sh — CHANGES_REQUESTED** [review]
see comments for timer decriptions

File: src/OXRS_Rack32.cpp (hunk 2 above)
Comment (sumnerboy12) [reply src/OXRS_Rack32.cpp:147]: `setting to 0 disables the timeout so last event always shown` - is that what you mean?

File: src/OXRS_Rack32.cpp (hunk 1 above)
Comment (sumnerboy12) [reply src/OXRS_Rack32.cpp:140]: Ok - will add that

File: src/OXRS_Rack32.cpp (hunk 2 above)
Comment (moinmoin-sh) [reply src/OXRS_Rack32.cpp:147]: yes

File: src/OXRS_Rack32.cpp (hunk 1 above)
Comment (sumnerboy12) [reply src/OXRS_Rack32.cpp]: @moinmoin-sh - pushed an update - changed the title and description - please review.

**Review by moinmoin-sh — APPROVED** [review]
Looks good. Can't test ATM

**Comment by sumnerboy12** [comment]
image

**Pull request merged**